### DISCUSSION TITLE
Smallest package

### DISCUSSION THREAD

**Discussion by gitra123** [termux/termux-packages#19209] [Q&A] [answered]
What is the smallest package in termux?

**Comment by TomJo2000** [comment, marked as answer]
That question doesn't really make any sense, but I'm guessing either:
- [`x11-repo`](https://github.com/termux/termux-packages/blob/master/packages/x11-repo/build.sh) (1204 Bytes),
- [`root-repo`](https://github.com/termux/termux-packages/blob/master/packages/root-repo/build.sh) (1220 Bytes)
or
- [`tur-repo`](https://github.com/termux/termux-packages/blob/master/packages/tur-repo/build.sh) (2884 Bytes)
Since they don't contain an *actual* package and are instead [stubs](https://en.wikipedia.org/wiki/Method_stub) that add their respective repo to the repo list.

As for the smallest *actual* package, idk probably something like [`yadm`](https://github.com/termux/termux-packages/blob/master/packages/yadm/build.sh) at ~28kB (if you ignore its dependency on Bash which comes preinstalled).

As for the smallest "standalone" package, probably [`tinyscheme`](https://github.com/termux/termux-packages/blob/master/packages/tinyscheme/build.sh) at ~30kB.
